### ISSUE TITLE
Duplicate rolls

### ISSUE THREAD

**Issue by bhollis** [48klocs/dim-wish-list-sources#32]
There are over 9000 duplicate rolls in the voltron.txt file that get dropped when the file is parsed. It'd be great to filter those out ahead of time as part of generating the overall file.

**Comment by whyayala** [comment]
I disagree. While I believe dim will only use the first match it finds in the file, having context of the number of times a roll was recommended can be meaningful. I am working on parsing out the voltron file to split it into seperate lists for mkb and controller users. I also want to be able to apply a "weight" to rolls signifying how many times they occured and include that info in the wishlist notes. I think that could provide more insight to players on how optimal their roll is.

repo here: https://github.com/whyayala/dim-wishlist-cleanup
